src/core/edit/edit.cell.view.js

```diff
@@ -49,9 +52,17 @@ export class EditCellView {
 					}
 					else {
 						cell = model.navigation().cell;
+						const code = this.shortcut.keyCode;
+						if (code && code.length > 0) {
+							const reserved = Array.from(this.shortcut.codeMap.values()).some(val => val === code);
+							if (!reserved) {
+								this.memory = code;
```

Comment:
why u need to know it ? it's not allowed to cell.value until commit is triggred, for that this.value was introduced